vertical-pod-autoscaler/enhancements/3919-customized-recommender-vpa/README.md

```diff
@@ -98,18 +99,32 @@ validation:
        type: object
        required: []
        properties:
-         recommenderName:
-           type: string
+         recommenders:
+           type: array
+           items:
+             properties:
+               name:
```

Comment:
Could you please comment (preferably in the KEP itself) why a struct?
I know it's forward-looking but maybe there already is a use-case that can support this choice?

Reply:
Done.